game/game.py

```diff
@@ -90,4 +76,28 @@ def make_decision(user_choice, comp_choice):
     return winner
 
 
+def print_report():
```

Comment:
this function, shouldn't have logic at the beginning, so we don't need if conditions.
I wanted it to do just one call of print() and print should have multiline string with whole report
Also this functions should take many arguments, like comp_choice, user_choice, winner, score
Also why you moved here all function calls and variables, put it back